stable/aws-for-fluent-bit/README.md

```diff
@@ -39,7 +39,7 @@ helm delete aws-for-fluent-bit --namespace kube-system
 | `service.parsersFiles` | List of available parser files | `/fluent-bit/parsers/parsers.conf` |
 | `service.extraParsers` | Adding more parsers with this value | `""` |
 | `input.*` | Values for Kubernetes input | |
```

Comment:
nit: Can you fix the language above too? This is the kubernetes tail log file input.